packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
 
 # Layout transitions
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+Providing a layout transition into an UI element alters each layout change into a smooth transition. A layout change may consist of both changes of size and position and both of them can be animated with layout transition.
```

Comment:
Before previous changes from Latropos, it was something like this:
```Layout transition allows you to animate changes in the layout of elements.```
Is it better?

Reply:
IDK, but if you want to rephrase it simpler, you can try this direction:

1. Providing a layout transition into an UI element alters each layout change into a smooth transition.
⬇️
Layout transitions replace **layout changes** with smooth transitions.

2. Layout change may consist of both changes of size and position and both of them can be animated with layout transition.
⬇️
Each layout change may include changes of **size** and **position** and both of them can be animated.

Also using bold to highlight most important parts could be helpful. I really like the summary to mention that layout change include size, since it is not obvious and our entering & existing animations have nothing to do with size

Reply:
So maybe something like that?

Layout transitions allows you to replace layout changes with smooth transitions. Each layout change may include changes of size and position and both of them can be animated.